frontend/app/src/scenes/Home/scenes/UpsertCategory/index.tsx

```diff
@@ -63,6 +83,15 @@ class UpsertCategory extends React.Component<
 
   onFinish = (values: any) => {
     let statusCode: number, method: Method, url: string;
+    const translations: Translation[] = [];
+    // Creating the translation array to send it with the payload
+    Array.from(this.state.translations.entries()).map((language) => {
```

Comment:
the variable name should be changed. It's not providing a `language`

Reply:
What about this cuz it gives the both `translationKeyValuePair`

Reply:
what about `entry` or `translationEntry`?